packages/host/app/components/operator-mode/stack.gts

```diff
@@ -39,14 +41,16 @@ export default class OperatorModeStack extends Component<Signature> {
     }
 
     // add closing animation on last item
-    const lastItemIndex = this.args.stackItems.length - 1;
-    const lastItemEl = document.querySelector(
-      `[data-stack-card-index="${lastItemIndex}"]`,
-    ) as HTMLElement;
-    if (lastItemEl) {
-      lastItemEl.classList.add('closing-animation');
+    if (!isTesting()) {
+      const lastItemIndex = this.args.stackItems.length - 1;
+      const lastItemEl = document.querySelector(
+        `[data-stack-card-index="${lastItemIndex}"]`,
+      ) as HTMLElement;
```

Comment:
Ugh, I just noticed this. @hockjootan we really should not be doing querySelector outside of tests. Instead a modifier is a much better choice here